grouper/graph.py

```diff
@@ -718,17 +718,17 @@ def get_user_details(self, username, cutoff=None, expose_aliases=True):
                     "rolename": GROUP_EDGE_ROLES[role],
                 }
 
-                for permission in self.permission_metadata[parent_name]:
+                for service_permission in self.permission_metadata[parent_name]:
```

Comment:
What's the `service` in `service_permission` referring to?

Reply:
"service account". The service account permission grants are a different data type than group permission grants, so need to be stored in a separate variable. The variable naming here is not the greatest; I'm hoping to clean that up in a later diff once I've simplified this code some more.

Reply:
Huh, I'm getting lost somewhere in the code -- is `permission_metadata` not explicitly for users and not service accounts? My impression is that the rename should be on https://github.com/rra/merou/blob/c3f022fbf320da505453e2fd985190243789c51e/grouper/graph.py#L675 rather than this loop

Reply:
Oh, whoops, you're entirely correct, and I changed the wrong loop. Will fix in my next diff.